analysis/import/SonarImporter/src/test/kotlin/de/maibornwolff/codecharta/importer/sonar/dataaccess/SonarVersionAPIDatasourceIntegrationTest.kt

```diff
@@ -0,0 +1,87 @@
+package de.maibornwolff.codecharta.importer.sonar.dataaccess
+
+import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.get
+import com.github.tomakehurst.wiremock.client.WireMock.stubFor
+import com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo
+import com.github.tomakehurst.wiremock.junit.WireMockRule
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import java.net.MalformedURLException
+import java.net.URL
+import javax.ws.rs.core.MediaType
+
+class SonarVersionAPIDatasourceIntegrationTest {
+
+    @Rule
+    @JvmField
+    var wireMockRule = WireMockRule(PORT)
+    private val versionAPIDatasource = SonarVersionAPIDatasource("somename", createBaseUrl())
+
+    @Test
+    fun `should parse the version to 6 point 6`() {
+        stubFor(
+            get(urlEqualTo(URL_PATH))
+                .willReturn(
+                    aResponse()
+                        .withHeader("Content-Type", MediaType.TEXT_PLAIN + "; charset=utf-8")
+                        .withStatus(200)
+                        .withBody("6.5.3.1234")
+                )
+        )
+
+        val version = versionAPIDatasource.getSonarqubeVersion()
+
+        assertEquals(version.major, 6)
+        assertEquals(version.minor, 5)
+    }
+
+    @Test
+    fun `should fallback to default version if version is not parsable`() {
+        stubFor(
+            get(urlEqualTo(URL_PATH))
+                .willReturn(
+                    aResponse()
+                        .withHeader("Content-Type", MediaType.TEXT_PLAIN + "; charset=utf-8")
+                        .withStatus(200)
+                        .withBody("6")
+                )
+        )
+
+        val version = versionAPIDatasource.getSonarqubeVersion()
+
+        assertEquals(version.major, SonarVersionAPIDatasource.DEFAULT_VERSION.major)
+        assertEquals(version.minor, SonarVersionAPIDatasource.DEFAULT_VERSION.minor)
```

Comment:
I am fine with this behavior but in general: I would expect this to equal v6.0.0.0.

Reply:
done